dashboard/app/controllers/report_abuse_controller.rb

```diff
@@ -20,7 +20,7 @@ def report_abuse
                 "URL: #{params[:abuse_url]}",
                 "abuse type: #{params[:abuse_type]}",
                 "user detail:",
-                "#{params[:abuse_detail]}"
+                params[:abuse_detail]
```

Comment:
Though I can't guarantee that `params[:abuse_detail]` isn't `nil`, if it is `nil`, nothing will happen (`body` won't end with a blank line).